packages/react-liveness/src/components/FaceLivenessDetector/service/utils/TelemetryReporter/TelemetryReporter.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  StartFaceLivenessSessionCommandInput,
+  StartFaceLivenessSessionCommandOutput,
+} from '@aws-sdk/client-rekognitionstreaming';
+import { HttpRequest } from '@smithy/protocol-http';
+import {
+  BuildHandler,
+  BuildHandlerArguments,
+  BuildHandlerOutput,
+} from '@smithy/types';
+
+const DEFAULT_ATTEMPT_COUNT_TIMEOUT = 300000; // 5 minutes / 300000 ms
+
+export class TelemetryReporter {
+  static attemptCount = 0;
+  static timestamp = Date.now();
```

Comment:
Switched to static class vars here, this works for unmount and mount but does not work for refresh. What do you think?